src/Uno.UWP/Data/Pdf/PdfDocument.Droid.cs

```diff
@@ -0,0 +1,81 @@
+#if __ANDROID__
+using System;
+using System.Threading.Tasks;
+using Android.Graphics.Pdf;
+using Uno;
+
+namespace Windows.Data.Pdf
+{
+	public partial class PdfDocument : IDisposable
```

Comment:
Please make sure that the .editorconfig file is used, so that it aligns with the global whitespace configuration.

Reply:
I couldn't find it in Android solution filter

Reply:
Ok, I applied Code Cleanup, but for some reason it removes whitespace before `#if` conditions